docs/axes/cartesian/timeseries.md

```diff
@@ -0,0 +1,94 @@
+# TimeSeries Cartesian Axis
```

Comment:
Could we find a way to avoid this doc duplication: maybe merge both in the same page (`docs/axes/cartesian/time.md`) with a special section about the `TimeSeries` listing differences? or a common page for time base scales?